skimage/transform/tests/test_seam_carving.py

```diff
@@ -15,6 +15,7 @@ def test_seam_carving():
     testing.assert_allclose(out, 0)
 
     img = img.T
+    energy = energy.T
```

Comment:
Cheers @soupault. I agree that more tests are a good idea.  I'm happy work on that and will try and write something a bit more thorough tomorrow.